tools/docker/supabase-all-in-one/src/run-supabase.sh

```diff
@@ -6,13 +6,13 @@ do
   sleep 3
 done
 
-# Get list of images from docker-compose
-for img in `ls ${FREEZE_TMP_DIR}`
-do
-  echo "Load frozen image '${img}'"
-  tar -cC "${FREEZE_TMP_DIR}/${img}" . | docker load
-done
+# # Get list of images from docker-compose
+# for img in `ls ${FREEZE_TMP_DIR}`
+# do
+#   echo "Load frozen image '${img}'"
+#   tar -cC "${FREEZE_TMP_DIR}/${img}" . | docker load
+# done
 
```

Comment:
@rwat17 If this code is no longer needed and there is no specific reason to keep it around as a comment (e.g. it's not actual documentation) then I'd recommend deleting it just to make the rest of the code read easier.

Reply:
There is a problem with supabase tool we have in repo. Freeze script or old version of supabase is crashing the container. I need to investigate this but this process is pulling to much images, docker hub blocks me and i don't really have many chances to test this hah. This code just slipped under my eye, probably pushed one of the test branch finally. When I get to work tomorrow, I'll flip this.

Reply:
@petermetz For me itwasn't about migrations but building itself. Containers were randomly crashing for me and I have had no idea why. First I thought the cause is that I switched from Vbox to HyperV because our company requested that and this is a problem with networking or sth. But after it, I triend building it on my Private Macbook M1 it crashed too. I need to talk about this with outSH and fix it because this is crucial for front end development in our project :).
Pushed that little fix BTW

Reply:
@rwat17 That's real interesting, thanks for sharing all those details! Also thank you for doing the update!